ci/ci/github.py

```diff
@@ -1031,15 +1078,15 @@ async def deploy(self, db: Database, batch_client, steps, excluded_steps=()):
                 }
             )
             config.build(deploy_batch, self, scope='dev')
-            deploy_batch = await deploy_batch.submit()
+            await deploy_batch.submit()
```

Comment:
There's another instance on line 979 that looks like it also needs `deploy_batch =` removed from it.

Reply:
I've got no real understanding of what's going on with this code, of course 😄